### PR TITLE
[PM-11503] Organization Automatic Sync verbiage is misleading

### PR DESCRIPTION
**Pull request by jonashendrickx**
## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/web/src/locales/en/messages.json [hunk 1]
```diff
@@ -6041,15 +6041,15 @@
...
-  },
   "billingSyncKeyDesc": {
     "message": "A billing sync token from your cloud organization's subscription settings is required to complete this form."
   },
   "billingSyncKey": {
     "message": "Billing sync token"
   },
+  "billingTokenSyncDesc": {
+    "message": "Billing sync unlocks Families sponsorships and automatic license syncing on your server. After making updates in the Bitwarden cloud server, select Sync License to apply changes."
+  },
```
Comment (djsmith85) [inline apps/web/src/locales/en/messages.json:6052]: ⚠️  This has no changes to the removed `billingSyncDesc `

File: apps/web/src/app/billing/organizations/organization-subscription-selfhost.component.html [hunk 2]
```diff
@@ -121,7 +121,7 @@ <h2 class="mt-5">
         >
           <bit-label>{{ "manualUpload" | i18n }}</bit-label>
           <bit-hint>
-            {{ "manualUploadDesc" | i18n }}
+            {{ "manualBillingTokenUploadDesc" | i18n }}
```
Comment (djsmith85) [inline apps/web/src/app/billing/organizations/organization-subscription-selfhost.component.html:124]: ⚠️ `manualBillingTokenUploadDesc` is not defined in the `en/messages.json`

File: apps/web/src/app/billing/organizations/organization-subscription-selfhost.component.html [hunk 3]
```diff
@@ -90,7 +90,7 @@ <h2 class="mt-5">
             </a>
           </bit-label>
           <bit-hint>
-            {{ "billingSyncDesc" | i18n }}
+            {{ "automaticBillingSyncDesc" | i18n }}
```
Comment (djsmith85) [inline apps/web/src/app/billing/organizations/organization-subscription-selfhost.component.html:93]: ⚠️ `automaticBillingSyncDesc ` is not defined in the `en/messages.json`

**Review by djsmith85 — CHANGES_REQUESTED** [review]

File: apps/web/src/app/billing/organizations/organization-subscription-selfhost.component.html (hunk 3 above)
Comment (djsmith85) [reply apps/web/src/app/billing/organizations/organization-subscription-selfhost.component.html:93]: ⚠️  Still an issue. As the message belonging to `automaticBillingSyncDesc` has no changes over `billingSyncDesc`, maybe just revert this line and also the changes in `en/messages.json` where `billingSyncDesc` is removed.

File: apps/web/src/locales/en/messages.json (hunk 1 above)
Comment (djsmith85) [reply apps/web/src/locales/en/messages.json:6052]: ⚠️ Also still an issue

**Review by cturnbull-bitwarden — DISMISSED** [review]

File: apps/web/src/locales/en/messages.json (hunk 1 above)
Comment (djsmith85) [reply apps/web/src/locales/en/messages.json:6052]: This message key is defined, but not used anywhere. 

❓  Is it possible that `billingSyncDesc`was supposed to be replaced with `billingTokenSyncDesc`, but you then introduced `automaticBillingSyncDesc`as the final replacement? If my assumption is correct, then please remove `billingTokenSyncDesc`from `en/messages.json`

**Review by djsmith85 — CHANGES_REQUESTED** [review]

**Review by djsmith85 — DISMISSED** [review]

**Review by cturnbull-bitwarden — APPROVED** [review]

**Review by djsmith85 — APPROVED** [review]

**Review by amorask-bitwarden — APPROVED** [review]

**Pull request merged**